<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fury</class>
 <widget class="QWidget" name="Fury">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="layoutFury">
   <item>
    <widget class="QGroupBox" name="gbInput">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QGridLayout" name="layoutFiles">
      <item row="2" column="0">
       <widget class="QLabel" name="lbResolution">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Resolution</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbSample">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Sample</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="MantidQt::MantidWidgets::DataSelector" name="dsInput" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="loadLabelText" stdset="0">
         <string>Plot Input</string>
        </property>
        <property name="workspaceSuffixes" stdset="0">
         <stringlist>
          <string>_red</string>
          <string>_sqw</string>
         </stringlist>
        </property>
        <property name="fileBrowserSuffixes" stdset="0">
         <stringlist>
          <string>_red.nxs</string>
          <string>_sqw.nxs</string>
         </stringlist>
        </property>
        <property name="showLoad" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="MantidQt::MantidWidgets::DataSelector" name="dsResolution" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoLoad" stdset="0">
         <bool>true</bool>
        </property>
        <property name="workspaceSuffixes" stdset="0">
         <stringlist>
          <string>_res</string>
          <string>_red</string>
          <string>_sqw</string>
         </stringlist>
        </property>
        <property name="fileBrowserSuffixes" stdset="0">
         <stringlist>
          <string>_res.nxs</string>
          <string>_red.nxs</string>
          <string>_sqw.nxs</string>
         </stringlist>
        </property>
        <property name="showLoad" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="loFury">
     <item>
      <layout class="QVBoxLayout" name="properties"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="plot"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOutput">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QCheckBox" name="ckVerbose">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Verbose</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_0">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="ckPlot">
        <property name="text">
         <string>Plot Result</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="ckSave">
        <property name="text">
         <string>Save Result</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::DataSelector</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/DataSelector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
